---
layout: pageZH
title: Publications
background: grey
---
<div class="col-lg-12" style="padding-left:0;">
	<h2 class="section-heading text-uppercase">已发表论文</h2>
	<p class="section-sheading" style='font-size:1.2rem;'># 共同第一作者; * 共同通讯作者</p>
</div>

<div class='pub' id='pub-2021'>
	<p class='pub-year'>2021</p>

	<ul style='list-style: disc;' class='pub-text'>
		<li>
			Wang H#, Feng C#, Lu M, Zhang B, Xu Y, Zeng Q, Xi J, Zhou J, Ying X, <b>Zhang J*</b>, Yue W*, Pei X*. Integrative single-cell transcriptome analysis reveals a subpopulation of fibroblasts associated with favorable prognosis of liver cancer patients[J]. <i>Translational Oncology</i>, 2021, 14(1): 100981.
			<hr />
		</li>
	</ul>
</div>
<div class='pub' id='pub-2020'>
	<p class='pub-year'>2020</p>

	<ul style='list-style: disc;' class='pub-text'>
		<li>
			Chu S H, Chabon J R, Matovina C N, Minehart J C, Chen B-R, <b>Zhang J</b>, Kumar V, Xiong Y, Callen E, Hung P J, Feng Z, Koche R P, Liu X S, Chaudhuri J, Nussenzweig A, Sleckman B P, Armstrong S A. Loss of H3K36 Methyltransferase SETD2 Impairs V(D)J Recombination during Lymphoid Development[J]. <i>iScience</i>, 2020, 23(3): 100941.
			<hr />
		</li>
		<li>
			Zhang H#, Liu L#, <b>Zhang J</b>, Chen J, Ye J, Shukla S, Qiao J, Zhan X, Chen H, Wu C J, Fu Y-X*;, Li B*;. Investigation of Antigen-Specific T-Cell Receptor Clusters in Human Cancers[J]. <i>Clinical Cancer Research</i>, 2020, 26(6): 1359–1371.
			<hr />
		</li>
	</ul>
</div>
<div class='pub' id='pub-2019'>
	<p class='pub-year'>2019</p>

	<ul style='list-style: disc;' class='pub-text'>
		<li>
			<b>Zhang J</b>, Hu X, Wang J, Sahu A D, Cohen D, Song L, Ouyang Z, Fan J, Wang B, Fu J, Gu S, Sade-Feldman M, Hacohen N, Li W, Ying X*;, Li B*;, Liu X S*;. Immune receptor repertoires in pediatric and adult acute myeloid leukemia[J]. <i>Genome Medicine</i>, 2019, 11(1): 73.
			<hr />
		</li>
		<li>
			<b>Zhang J</b>, Qian L, Wu J, Lu D, Yuan H, Li W, Ying X*;, Hu S*;. Up-regulation of FAM64A promotes epithelial-to-mesenchymal transition and enhances stemness features in breast cancer cells[J]. <i>Biochemical and Biophysical Research Communications</i>, 2019.
			<hr />
		</li>
		<li>
			Hu X#, <b>Zhang J</b>#, Wang J, Fu J, Li T, Zheng X, Wang B, Gu S, Jiang P, Fan J, Ying X, Zhang J, Carroll M C, Wucherpfennig K W, Hacohen N, Zhang F, Zhang P, Liu J S*;, Li B*;, Liu X S*;. Landscape of B cell immunity and related immune evasion in human cancers[J]. <i>Nature Genetics</i>, 2019, 51(3): 560–567.
			<hr />
		</li>

	</ul>
</div>
<div class='pub' id='pub-2018'>
	<p class='pub-year'>2018</p>

	<ul style='list-style: disc;' class='pub-text'>
	<li>
		Hu X#, <b>Zhang J</b>#, Liu J S, Li B*;, Liu X S*;. Evaluation of immune repertoire inference methods from RNA-seq data[J]. <i>Nature Biotechnology</i>, 2018, 36(11): 1034–1034.
		<hr />
	</li>
		<li>
			Cornwell M, Vangala M, Taing L, Herbert Z, Köster J, Li B, Sun H, Li T, <b>Zhang J</b>, Qiu X, Pun M, Jeselsohn R, Brown M, Liu X S, Long H W. VIPER: Visualization Pipeline for RNA-seq, a Snakemake workflow for efficient and complete RNA-seq analysis[J]. <i>BMC Bioinformatics</i>, 2018, 19: 135.
			<hr />
		</li>
		<li>
			Xing X, Guo J, Ding G, Li B, Dong B, Feng Q, Li S, <b>Zhang J</b>, Ying X, Cheng X, Guo T, Du H, Hu Y, Zhou T, Wang X, Li L, Li Q, Xie M, Li L, Gao X, Shan F, Li Z, Jia S, Wen X, Wang J, Ji J. Analysis of PD1, PDL1, PDL2 expression and T cells infiltration in 1014 gastric cancer patients[J]. <i>OncoImmunology</i>, 2018, 7(3): e1356144.
			<hr />
		</li>
	</ul>
</div>
<div class='pub' id='pub-2017'>
	<p class='pub-year'>2017</p>

	<ul style='list-style: disc;' class='pub-text'>
		<li>
			Hu S, Yuan H, Li Z, <b>Zhang J</b>, Wu J, Chen Y, Shi Q, Ren W, Shao N, Ying X. Transcriptional response profiles of paired tumor-normal samples offer novel perspectives in pan-cancer analysis[J]. <i>Oncotarget</i>, 2017, 8(25): 41334–41347.
			<hr />
		</li>
		<li>
			Li B, Li T, Wang B, Dou R, <b>Zhang J</b>, Liu J S, Liu X S. Ultrasensitive detection of TCR hypervariable-region sequences in solid-tissue RNA–seq data[J]. <i>Nature Genetics</i>, 2017, 49(4): 482–483.
			<hr />
		</li>
		<li>
			Wu J, Hu S, Chen Y, Li Z, <b>Zhang J</b>, Yuan H, Shi Q, Shao N, Ying X. BCIP: a gene-centered platform for identifying potential regulatory genes in breast cancer[J]. <i>Scientific Reports</i>, 2017, 7: 45235.
			<hr />
		</li>
		<li>
			Chen Y, Li Z, Hu S, <b>Zhang J</b>, Wu J, Shao N, Bo X, Ni M, Ying X. Gut metagenomes of type 2 diabetic patients have characteristic single-nucleotide polymorphism distribution in Bacteroides coprocola[J]. <i>Microbiome</i>, 2017, 5: 15.
			<hr />
		</li>
	</ul>
</div>
<div class='pub' id='pub-2016'>
	<p class='pub-year'>2016</p>

	<ul style='list-style: disc;' class='pub-text'>
		<li>
			Ren W, <b>Zhang J</b>, Li W, Li Z, Hu S, Suo J, Ying X. A Tumor-Specific Prognostic Long Non-Coding RNA Signature in Gastric Cancer[J]. <i>Medical Science Monitor</i>, 2016, 22: 3647–3657.
			<hr />
		</li>
		<li>
			Li T, Chen L, Cheng J, Dai J, Huang Y, <b>Zhang J</b>, Liu Z, Li A, Li N, Wang H, Yin X, He K, Yu M, Zhou T, Zhang X, Xia Q. SUMOylated NKAP is essential for chromosome alignment by anchoring CENP-E to kinetochores[J]. <i>Nature Communications</i>, 2016, 7: 12969.
			<hr />
		</li>
		<li>
			Li Z, Chen Y, Hu S, <b>Zhang J</b>, Wu J, Ren W, Shao N, Ying X. Integrative analysis of protein-coding and non-coding RNAs identifies clinically relevant subtypes of clear cell renal cell carcinoma[J].<i> Oncotarget</i>, 2016, 7(50): 82671–82685.
			<hr />
		</li>
	</ul>
</div>
